app/Console/Commands/UpdateSignupAndOrPostField.php

```diff
@@ -78,7 +78,9 @@ public function handle()
                         info('rogue:updatefield: changing ' . $targetField . ' to ' . $targetNewValue . ' for signup ' . $signup->id);
                     }
 
-                    $this->signups->update($signup, [$targetField => $targetNewValue]);
+                    // Update signup but don't log that it was sent to Quasar in interest of taking up too much space in Papertrail.
+                    // @TODO: when running updates for smaller amount of records, do we want to turn logs back on?
```

Comment:
You can take this param in as a flag in the command, so you can set the option when you run the command.

Reply:
`$log = $this->option('log');`

Reply:
oh great call, @sbsmith86 ! Just updated to that ! :)